library/src/main/java/eu/miaplatform/customplugin/springboot/CPController.java

```diff
@@ -22,6 +27,18 @@
     @Autowired
     protected CPService customPluginService;
 
+    @PostConstruct
+    public void init() {
+        try {
+            EnvConfiguration.parseEnvironment();
+        } catch (InvalidEnvConfigurationException ex) {
+            ex.printStackTrace();
+            ConfigurableApplicationContext ctx = new SpringApplicationBuilder(Application.class)
+                    .web(WebApplicationType.NONE).run();
+            SpringApplication.exit(ctx, () -> 0);
```

Comment:
Does that `0` mean that the process will exit with status `0` ? Maybe it'd be better to exit the process with status `1`